epcis/src/main/java/io/openepcis/model/epcis/modifier/CustomContextSerializer.java

```diff
@@ -10,7 +10,7 @@
 
 public class CustomContextSerializer extends JsonSerializer<List<Object>> {
 
-  private static DefaultJsonSchemaNamespaceURIResolver namespaceResolver =
+  private final DefaultJsonSchemaNamespaceURIResolver namespaceResolver =
```

Comment:
context is shared - please fix, each thread / request must be running on it's own context